.evergreen/run-perf-tests.sh

```diff
@@ -22,7 +22,7 @@ pip install simplejson
 
 python setup.py build_ext -i
 start_time=$(date +%s)
-python test/performance/perf_test.py
+python test/performance/perf_test.py --locals
```

Comment:
`--locals` is a unittest option: https://docs.python.org/3/library/unittest.html#cmdoption-unittest-locals
> Show local variables in tracebacks

This improves the tracebacks displayed when a test fails by including the local variables in each frame.

Reply:
Will this slow down the benchmarks?

Reply:
Nope. Here's the cpython change which added it: https://github.com/python/cpython/commit/f0c819acd0f#diff-ba8a6f32f39f6047ea998da6ddc7356a30eff06dd2b84d19f9a285c3162d2a93R187

The flag only changes the behavior of how tracebacks are printed after a test fails. Since we don't expect any tests to fail in the normal case there won't be any perf impact.